HISTORY.md

```diff
@@ -0,0 +1,420 @@
+# Old Changelog
```

Comment:
Is there a reason behind splitting the changelog into separate CHANGELOG.md and HISTORY.md files?

Reply:
The changelog generator I`m using completely overwrites the old changelog.
It has a feature where it can import and old changelog if it is placed in HISTORY.md